settings.gradle

```diff
@@ -3,4 +3,5 @@ include ':marklogic-data-hub'
 include ':ml-data-hub-plugin'
 include ':quick-start'
 
+include ':examples:data-integration-tests'
```

Comment:
@grechaw what's the advantage to adding this here?

Reply:
It made IDEA recognize that project.  However if it causes any issues at all then of course it should be removed; now that that example works from the command line, I don't think it's needed as an explicit subproject.